apps/trading/pages/_app.page.tsx

```diff
@@ -95,6 +97,7 @@ function AppBody({ Component }: AppProps) {
       <ToastsManager />
       <InitializeHandlers />
       <MaybeConnectEagerly />
+      <PartyData />
```

Comment:
Maybe instead adding blank component better would put it to  `<InitializeHandlers />` 2 lines above?